From c6db156881aea86ee947393c41f6910e68f7354b Mon Sep 17 00:00:00 2001
From: Maximilian Luz <luzmaximilian@gmail.com>
Date: Tue, 7 Jun 2022 02:22:58 +0200
Subject: [PATCH] Redirect file lookups to /var/lib/rmtfs

Signed-off-by: Maximilian Luz <luzmaximilian@gmail.com>
---
 rmtfs.service.in | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/rmtfs.service.in b/rmtfs.service.in
index 8384043..bc9858e 100644
--- a/rmtfs.service.in
+++ b/rmtfs.service.in
@@ -4,7 +4,7 @@ Requires=qrtr-ns.service
 After=qrtr-ns.service
 
 [Service]
-ExecStart=RMTFS_PATH/rmtfs -r -P -s
+ExecStart=RMTFS_PATH/rmtfs -r -s -o /var/lib/rmtfs
 Restart=always
 RestartSec=1
 
-- 
2.36.1

